util/settings/settings.go

```diff
@@ -45,6 +45,21 @@ import (
 	tlsutil "github.com/argoproj/argo-cd/v3/util/tls"
 )
 
+var CommitMessageTemplate = `{{ .metadata.subject }}
```

Comment:
```suggestion
var CommitMessageTemplate = `{{.metadata.drySha | trunc 7}}: {{ .metadata.subject }}
```

Sorry for the last-minute change, just remembered I wanted a sha prefix.